src/palakSJun20/ArrayUnion.java

```diff
@@ -0,0 +1,58 @@
+/*
+Program 1: Find union of two arrays.
+Input: arr1 -> {10.45, 14.0, 18.35, 88.88, 54.10, 18.35}
+       arr2 -> {17.20, 13.30, 10.45, 18.35, 84.33, 13.30}
+
+Output: 10.45, 14.0, 18.35, 88.88, 54.10, 17.20, 13.30, 84.33
+Method Signature : double[] getUnionOfArray(double[] arr1, double[] arr2) */
+package palakSJun20;
+
+import java.util.Arrays;
+
+public class ArrayUnion {
+	int count = 0;
+
+	void findArrayUnion(double[] array1, double[] array2) {
+		double[] tempArray3 = new double[array1.length + array2.length]; // length =12
+		
+		// Calling Method to find Unique Number Array
+		tempArray3 = findUniqueNumberArray(array1, tempArray3);
+		tempArray3 = findUniqueNumberArray(array2, tempArray3);
```

Comment:
double[] tempUnionArray = new double[arr1.length + arr2.length];
		tempUnionArray = findUniqueNumberArray(arr2, findUniqueNumberArray(arr1, tempUnionArray));

Just write like this.

Reply:
Done